core/geometry/src/curve/internalContexts/geometry2d/UnboundedParabola.ts

```diff
@@ -0,0 +1,276 @@
+    super();
+    this.pointA = pointA;
+    this.vectorU = vectorU;
+    this.vectorV = vectorV;
+  }
+    /**
+     * Return a clone of this Parabola.
+     */
+    public clone () : UnboundedParabola2d {
+      // (The create method clones the inputs . . .)
+      return  UnboundedParabola2d.createCenterAndAxisVectors (this.pointA, this.vectorU, this.vectorV);
+    }
+    /**
+     * Return a clone of this circle, with radius negated
+     */
+
+  /**
+   * Create an UnboundedParabola2d from an xy object and a radius.
+   * @param pointA xy coordinates of center
+   * @param vectorU vector from pointA to to theta=0 vertex, i.e. along axis on the "inside" of the curve
+   * @param vectorV vector from pointA to the transverse direction.
+   * @returns
+   */
+  public static createCenterAndAxisVectors(center: XAndY, vectorU: XAndY, vectorV: XAndY): UnboundedParabola2d {
+    return new UnboundedParabola2d(Point2d.create (center.x, center.y),
+          Vector2d.create (vectorU.x, vectorU.y),
+          Vector2d.create (vectorV.x, vectorV.y));
+  }
+  /**
+   * Return the local (uv) coordinate of a global point.
+   * @param spacePoint
+   */
+  public globalToLocal (spacePoint: XAndY):Vector2d | undefined{
+    const result = Vector2d.create ();
+    if (SmallSystem.linearSystem2d (
+      this.vectorU.x, this.vectorV.x,
+      this.vectorU.y, this.vectorV.y,
+      spacePoint.x - this.pointA.x, spacePoint.y - this.pointA.y, result))
+      return result;
+    return undefined;
+  }
+  /**
+   *
+   * @param radians parametric angle on the Parabola
+   * @returns xy coordinate on the Parabola
+   */
+  public override radiansToPoint2d (radians: number): Point2d | undefined{
+    const c = Math.cos (radians);
+    const s = Math.sin (radians);
+    const q = 1 + c;
+    if (Geometry.isSmallMetricDistance (Math.abs (q)))
+        return undefined;
+    return this.pointA.plus2Scaled (this.vectorU, s/q,  this.vectorV, (1 - c)/q);
+  }
+  /** OPTIONAL method to return a tangent at given radians value.
+ * * The default implementaiton returns undefined.
+ * * Concrete classes that can be expressed as a function of radians should implement this.
+ */
+    public override radiansToTangentVector2d (radians: number): Vector2d | undefined {
+      const c = Math.cos (radians);
+      const s = Math.sin (radians);
+      const q  = 1+c;
+      const u = Geometry.conditionalDivideCoordinate (1, q);
+      const v = Geometry.conditionalDivideCoordinate(2 * s, q * q);
+      if (u === undefined || v === undefined)
+        return undefined;
+      return Vector2d.createAdd2Scaled (this.vectorU, u, this.vectorV, v);
+    }
+
+  /**
+   * Return the implicit function value v - u*u
+   * @param xy space paoint
+   */
+public override functionValue (xy: XAndY) : number {
+  const vectorUV = this.globalToLocal (xy);
+  if (vectorUV === undefined)
+    return 0;
+  return vectorUV.y - vectorUV.x * vectorUV.x;
```

Comment:
are you sure this is not ` return vectorUV.y * vectorUV.y - vectorUV.x * vectorUV.x;`?